aos/blueprint.py

```diff
@@ -1686,6 +1687,8 @@ def create_security_zone(
         sz = self.create_security_zone_from_json(bp_id, sec_zone)
         logger.info(f"Creating Security-zone '{name}' in blueprint '{bp_id}'")
 
+        time.sleep(0.5)
```

Comment:
different environments will require a different delay here. The correct solution would be to implement a repeat_until on the sz create task being successful or a get_security_zone != NullSecurityZone.

I think I have the later used somewhere else.

Reply:
+1 to @that1guy15. Sleeps with magic numbers are only the source of confusion and more issues in the future.

Reply:
Yes, I discuss this today with that1guy15 - I agree that static sleep can be confusing and problem could occur even after 0.5sec.. so this problem should be handle by "repeat_until" which will not raise AosAPIError in case of 404

Reply:
Right. sleep must be removd

Reply:
Here is how we implement elsewhere:
`repeat_until(lambda:
                     self.get_security_zone(bp_id, sz["id"]) != NullSecurityZone,
                     timeout=timeout)`
                     
You can also capture or lookup the task ID on the sz create, then wait for the task to finish successful before moving on using get_task()